src/sentry/static/sentry/app/actionCreators/account.jsx

```diff
@@ -23,3 +23,10 @@ export function updateUser(user) {
   // Ideally we'd fire an action but this is gonna get refactored soon anyway
   ConfigStore.set('user', user);
 }
+
+export function logout() {
```

Comment:
I think we pass api client inconsistently atm, but a reason for having the caller (generally a component) pass in an API instance is so that the component can cancel ongoing requests when component unmounts.

Reply:
Ok. I can update this and work towards refactoring other uses to accept the client as a parameter as well.